src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/Snapshots/VsProjectProperties.cs

```diff
@@ -1,18 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
-
-using Microsoft.VisualStudio.ProjectSystem.PackageRestore;
-using NuGet.SolutionRestoreManager;
-
-namespace Microsoft.VisualStudio.ProjectSystem.VS.PackageRestore;
-
-/// <summary>
-///     Wraps a collection of <see cref="ProjectProperty"/> instances to implement the <see cref="IVsProjectProperties"/>
-///     interface for NuGet.
-/// </summary>
-internal class VsProjectProperties : ImmutablePropertyCollection<IVsProjectProperty, ProjectProperty>, IVsProjectProperties
```

Comment:
Nothing is using `ImmutablePropertyCollection<,>` anymore. As discussed offline, I'll push a commit that removes it.